config.yaml

```diff
@@ -1,4 +1,12 @@
 options:
+  debug:
+    type: boolean
+    default: False
+    description: |
+      Set it to enable LogLevel=debug and reload debug image for containers.
+      Access kube-multus for debug with:
+      kubectl exec -n <model-name> -it <pod-name> -- bash
+      $ dlv attach --continue <PID>
```

Comment:
I'm a bit confused about the usage of dlv here. What PID am I supposed to connect to? There is no persistent golang process:

```
$ kubectl exec -n k8s -it multus-jf8h4 -- ps aux
Defaulting container name to kube-multus.
Use 'kubectl describe pod/multus-jf8h4 -n k8s' to see all of the containers in this pod.
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.0  0.0   4100  3464 ?        Ss   20:00   0:00 /bin/bash /en
root          57  0.0  0.0   2508   592 ?        S    20:00   0:00 sleep infinit
root          64  0.0  0.0   5888  3068 pts/0    Rs+  20:06   0:00 ps aux
```